lib/models/mirror.js

```diff
@@ -15,9 +16,13 @@ var Mirror = new mongoose.Schema({
   },
   contact: {
     type: String,
-    ref: 'Contact'
+    ref: 'Contact',
+    required: true
```

Comment:
I've noticed that there are existing records without a contact for mirrors, which has been a mystery....